packages/parcel-runtime/src/utils/bgsw.ts

```diff
@@ -0,0 +1,53 @@
+import { extCtx, getHostname, getPort, runtimeData } from "./0-patch-module"
+
+const devServer = `${
+  runtimeData.secure ? "https" : "http"
+}://${getHostname()}:${getPort()}/`
+
+export async function pollingDevServer(delay = 1470) {
+  while (8) {
```

Comment:
i get it's a fun thing to do 8, but let's make this while(true)

Reply:
Boo